testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -520,4 +524,41 @@ public void testSelectInformationSchemaColumns()
                         "('tables'), " +
                         "('views')");
     }
+
+    @Test
+    public void testRenameTableAcrossSchema()
+    {
+        if (!hasBehavior(SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS)) {
+            if (!hasBehavior(SUPPORTS_RENAME_TABLE)) {
```

Comment:
I don't get it, I would like to remove this if.

Reply:
I had to add this to be able to make a more specific assert for the exception message. So I'd either have to fall back to a more generic assert or keep this.